GeoHealthCheck/plugins/probe/ogc3dtiles.py

```diff
@@ -0,0 +1,61 @@
+from GeoHealthCheck.probe import Probe
+import requests
+
+
+class OGC3DTiles(Probe):
+    """
+    OGC3DTiles
+    """
+
+    NAME = 'GET Tileset.json and tile data'
+    DESCRIPTION = 'OGC3DTiles'
+    RESOURCE_TYPE = 'OGC:3DTiles'
+    REQUEST_METHOD = 'GET'
+
+    CHECKS_AVAIL = {
+        'GeoHealthCheck.plugins.check.checks.HttpStatusNoError': {
+            'default': True
+        },
+    }
+    """Checks avail"""
+
+    def perform_request(self):
+        url_base = self._resource.url
+
+        # Remove trailing '/' if present
+        if url_base.endswith('/'):
+            url_base = url_base[:-1]
+        elif url_base.endswith('/tileset.json'):
+            url_base = url_base.split('/tileset.json')[0]
+
+        # Request tileset.json
+        try:
+            tile_url = url_base + '/tileset.json'
+            self.log('Requesting: %s url=%s' % (self.REQUEST_METHOD, tile_url))
+            self.response = Probe.perform_get_request(self, tile_url)
+            self.run_checks()
+        except requests.exceptions.RequestException as e:
+            msg = "Request Err: Error requesting tileset.json %s %s" \
+                % (e.__class__.__name__, str(e))
+            self.result.set(False, msg)
+
```

Comment:
Better add `return` here as `/tileset.json` already gives `Exception`, no use going on.

Reply:
You're right. Added return here.

Reply:
This was actually a mistake where the probe would only look directly under the root for a data url. I changed it to use recursion to "dig deeper" into the tileset.json to find the tile data url. The url you found https://3dbag.nl/download/3dtiles/v210908_fd2cee53/lod22/tileset.json does now give back a "True" result.